Rethinking_2/Chp_02.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>For the second edition, I propose the following import-statements convention, so that they're consistent across notebooks:</p><ol><li>Python core libraries first (<code>import math</code>)</li><li>Absolute imports of external packages (<code>import arviz as az</code>)</li><li>Relative imports of external packages (<code>from arviz import plot_trace</code>)</li><li>All packages are alphabetically ordered within those three groups.</li></ol><p> </p><p>Here that would be:</p><p> </p><p><code>import arviz as az</code></p><p><code>import matplotlib.pyplot as plt</code></p><p><code>import numpy as np</code></p><p><code>import pymc3 as pm</code></p><p><code>import scipy.stats as stats</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>